Source/Game/Game/Editor/EaseCurveTool.cpp

```diff
@@ -0,0 +1,402 @@
+#include "EaseCurveTool.h"
+
+#include <Base/Math/Interpolation.h>
+
+#include <imgui/imgui.h>
+
+namespace Editor
+{
+    EaseCurveTool::EaseCurveTool()
+        : BaseEditor(GetName(), false)
+    {
+
+    }
+
+    void EaseCurveTool::DrawImGui()
+    {
+        if (ImGui::Begin(GetName(), NULL, ImGuiWindowFlags_AlwaysAutoResize))
+        {
+            if (OpenMenu("Settings"))
+            {
+                ImGui::Checkbox("2D Preview", &_preview2D);
+                ImGui::Checkbox("1D Preview", &_preview1D);
+
+                CloseMenu();
+            }
+
+            static bool useInverse = true;
```

Comment:
I'm not a fan of all of these static/const variables being defined within the DrawImGui function, perhaps we could have them stored in the class?